### PR TITLE
Adding featureflags to .metamaskrc.dist file

### PR DESCRIPTION
**Pull request by jpuri**
Adding featureflags to .metamaskrc.dist file

### CODE REVIEW

**Review by segun — DISMISSED** [review]

File: .metamaskrc.dist [hunk 1]
```diff
@@ -2,3 +2,5 @@
 PASSWORD=METAMASK PASSWORD
 INFURA_PROJECT_ID=00000000000
 SEGMENT_WRITE_KEY=
+ONBOARDING_V2=1
```
Comment (Gudahtt) [inline .metamaskrc.dist:5]: Should these default to being enabled? Right now they're disabled by default.

Comment (jpuri) [reply .metamaskrc.dist:5]: in dev mode it will be nice to have these enabled.

Comment (Gudahtt) [reply .metamaskrc.dist:5]: I would have found it surprising personally. Unless you're working on one of these features, you wouldn't have reason to enable them. We're still maintaining the old UI after all, fixing bugs and making other improvements.

Comment (jpuri) [reply .metamaskrc.dist]: Make sense, I removed the value to disable the feature flag

**Review by Gudahtt — DISMISSED** [review]
LGTM! Though two other flags that might be worth including are `SWAPS_USE_DEV_APIS` and `COLLECTIBLES_V1`.

**Comment by darkwing** [comment]
New flags added

**Review by Gudahtt — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**